### PR TITLE
Add instructions on how to specify customized Configuration [SAME VERSION]

### PR DESCRIPTION
**Pull request by jiria**
**What this PR does / why we need it**:
Point to modifying-akri-installation doc also for deploying customized Akri configuration. This will help folks that have more complicating pod spec to specify custom configuration.

**Special notes for your reviewer**:

**If applicable**:
- [x] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] added code adheres to standard Rust formatting (`cargo fmt`)
- [ ] code builds properly (`cargo build`)
- [ ] code is free of common mistakes (`cargo clippy`)
- [ ] all Akri tests succeed (`cargo test`)
- [ ] inline documentation builds (`cargo doc`)
- [ ] version has been updated appropriately (`./version.sh`)

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -113,6 +113,6 @@ helm install akri akri-helm-charts/akri
     watch kubectl get pods,services,akric,akrii -o wide
     ```
 
-### Modifying your Akri installation
+### Modifying your Akri installation or deploying customized Akri Configuration
```
Comment (bfjelds) [inline docs/user-guide.md:116]: maybe "deploying **a** customized"

**Review by bfjelds — APPROVED** [review]

**Comment by kate-goldenring** [comment]
Should the name of the modifying your akri installation doc also be changed to incorporate custom configuration as well?

**Comment by jiria** [comment]
Perhaps. Happy to change if you have a proposal to use.

We might also want to bubble it up a bit in the modifying doc. Thoughts?

**Comment by kate-goldenring** [comment]
> Perhaps. Happy to change if you have a proposal to use.
> 
> We might also want to bubble it up a bit in the modifying doc. Thoughts?

Yeah. I think that document can be reorganized. The second bullet could be "Generating, modifying and applying a custom configuration" and then the [modifying a deployed configuration](https://github.com/deislabs/akri/blob/main/docs/modifying-akri-installation.md#modifying-a-deployed-configuration) section could reference it instead of consuming it.

**Comment by jiria** [comment]
Ok, tried to incorporate the changes you proposed in the latest iteration. Let me know what you think.

**Comment by kate-goldenring** [comment]
> Ok, tried to incorporate the changes you proposed in the latest iteration. Let me know what you think.

This is great! Thanks @jiria. Could you possibly rename the file to be customizing-akri-installation.md to be consistent with your changes?

**Review by kate-goldenring — APPROVED** [review]

File: docs/onvif-configuration.md [hunk 2]
```diff
@@ -91,8 +91,8 @@ By default, the generic ONVIF Configuration will create services for all the bro
 
 ## Modifying a Configuration
 More information about how to modify an installed Configuration, add additional protocol Configurations to a cluster, or
-delete a Configuration can be found in the [Modifying a Akri Installation
-document](./modifying-akri-installation.md).
+delete a Configuration can be found in the [Customizing Akri Installation
```
Comment (jiria) [inline docs/onvif-configuration.md:94]: @kate-goldenring @bfjelds , should all these by Modifying _an_ Akri installation and if so, should I rename the docs to match?

**Pull request merged**